libpit/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.4)
 project(libpit)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Dnullptr=NULL")
```

Comment:
This seems to break compilation on linux and windows according to the newly introduced github action

Reply:
Using the changes from [e.foundation](https://gitlab.e.foundation/e/tools/heimdall/-/commit/af5c83a45c8580c3c264db008cd0582c0cf1b757) is cleaner than the changes suggested here, since they don't break compilation for other platforms. This works and compiles well on [my fork](https://github.com/amo13/Heimdall):